src/components/header/authorizedUser/styles.css

```diff
@@ -0,0 +1,65 @@
+.menu {
+  position: relative;
+  padding: 0 0 15px;
+  transform: translate(-127px, 0);
+}
+
+.dropDown {
+  background: rgba(93 93 93 / 40%);
+  flex-direction: column;
+  padding: 7px 17px 0;
+  min-width: 90px;
+  position: absolute;
+  align-items: center;
+  top: 33px;
+  display: none;
+  left: 50%;
+  transform: translate(-50%, 0);
+  list-style: none;
+  margin: 0;
+  border-radius: 15px;
+}
+
+.dropDown a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.dropDown a:hover {
+  text-decoration: underline 1px;
+  text-underline-offset: 5px
+}
+
+.dropDown li {
+  font-size: 26px;
+  line-height: 24px;
```

Comment:
how comes line height is less then font size?

Reply:
This is mistake. There should generally be a font-size of 25 pixels